cloud/aws/bin/pgadmin.py

```diff
@@ -0,0 +1,107 @@
+import os
+import sys
+import time
+import re
+import urllib.error
+import urllib.request
+
+from cloud.aws.templates.aws_oidc.bin import resources
+from cloud.aws.templates.aws_oidc.bin.aws_cli import AwsCli
+from cloud.shared.bin.lib import terraform
+
+
+def run(config):
+    os.environ["TF_VAR_pgadmin_cidr_allowlist"] = get_cidr_list()
+    os.environ["TF_VAR_pgadmin"] = "true"
+
+    if not terraform.perform_apply(config):
+        sys.stderr.write("Terraform deployment failed.")
+        raise ValueError("Terraform deployment failed.")
+
+    pgurl = f"{config.get_base_url()}:4433"
+    print(
+        "\npgadmin terraform deployment finished. Waiting for pgadmin service (some request failures are expected):"
+    )
+    wait_for_pgadmin_response(pgurl)
+
+    print(f"\npgadmin service is available. URL: {pgurl}")
+    print_secrets(config)
+
+    input(
+        "\nWARNING: it is strongly recommended to clean up pgadmin resources once they are no-longer needed.\n"
+        "Run 'bin/deploy --tag=latest' to manually clean up pgadmin resources.\n\n"
```

Comment:
Now I realize that this is potentially problematic. If they run `bin/deploy --tag=latest` then they will deploy latest civiform which should never be used in prod. Instead they should deploy whatever civiform they already have running.

Reply:
Updated the output to use whatever --tag the user passed to /bin/run